<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Browser Launched Successfully]]>
    </line>
    <line>
      <![CDATA[logged in Successfully]]>
    </line>
    <line>
      <![CDATA[logged out successfully]]>
    </line>
    <line>
      <![CDATA[logged in Successfully]]>
    </line>
    <line>
      <![CDATA[logged out successfully]]>
    </line>
    <line>
      <![CDATA[browser closed successfully]]>
    </line>
    <line>
      <![CDATA[database disconnected]]>
    </line>
  </reporter-output>
  <suite started-at="2025-09-26T17:47:46 IST" name="Default suite" finished-at="2025-09-26T17:48:28 IST" duration-ms="41782">
    <groups>
    </groups>
    <test started-at="2025-09-26T17:47:46 IST" name="Default test" finished-at="2025-09-26T17:48:28 IST" duration-ms="41782">
      <class name="com.Vtiger.Organization.CreateOrganizationTest">
        <test-method is-config="true" signature="databaseConnection()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:47:46 IST" name="databaseConnection" finished-at="2025-09-26T17:47:46 IST" duration-ms="24" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- databaseConnection -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:47:47 IST" name="openBrowser" finished-at="2025-09-26T17:47:50 IST" duration-ms="3614" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Browser Launched Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="loginToVtiger()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:47:50 IST" name="loginToVtiger" finished-at="2025-09-26T17:47:55 IST" duration-ms="4856" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged in Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginToVtiger -->
        <test-method signature="createOrg()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:47:55 IST" name="createOrg" finished-at="2025-09-26T17:48:14 IST" duration-ms="18567" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createOrg -->
        <test-method is-config="true" signature="logoutFromVtiger()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:48:14 IST" name="logoutFromVtiger" finished-at="2025-09-26T17:48:16 IST" duration-ms="2666" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged out successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- logoutFromVtiger -->
        <test-method is-config="true" signature="loginToVtiger()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:48:16 IST" name="loginToVtiger" finished-at="2025-09-26T17:48:21 IST" duration-ms="4466" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged in Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginToVtiger -->
        <test-method signature="intentionalFail()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:48:21 IST" name="intentionalFail" finished-at="2025-09-26T17:48:21 IST" duration-ms="13" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at com.Vtiger.Organization.CreateOrganizationTest.intentionalFail(CreateOrganizationTest.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intentionalFail -->
        <test-method is-config="true" signature="logoutFromVtiger()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:48:21 IST" name="logoutFromVtiger" finished-at="2025-09-26T17:48:28 IST" duration-ms="6567" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logged out successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- logoutFromVtiger -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:48:28 IST" name="closeBrowser" finished-at="2025-09-26T17:48:28 IST" duration-ms="357" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[browser closed successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="databasedisconnection()[pri:0, instance:com.Vtiger.Organization.CreateOrganizationTest@757277dc]" started-at="2025-09-26T17:48:29 IST" name="databasedisconnection" finished-at="2025-09-26T17:48:29 IST" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[database disconnected]]>
            </line>
          </reporter-output>
        </test-method> <!-- databasedisconnection -->
      </class> <!-- com.Vtiger.Organization.CreateOrganizationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
